### ISSUE TITLE
Markers Clusters Disappear on Zoom/Fit

### ISSUE THREAD

**Issue by Life4Cali** [angular-ui/angular-google-maps#1047]
So in version 2.0.11 everything worked well, marker clustering worked and refreshed when zooming and fitting to map (even with animations on the markers).

Now as of 2.0.12 if you try to zoom or have the map fit, the clusters disappear and markers do not take their place as you zoom in.  I've tried custom clusters, default clusters, both with or without custom markers; no luck.

I'm not sure if I saw an issue like this already but it seemed like it was closed and I've updated to the latest version since then.  I figured I'd specify that for me 2.0.11 worked fine, then 2.0.12 failed.

**Comment by nmccready** [comment]
Clustering issues have been fixed on master, check against that

**Comment by normank86** [comment]
@nmccready thanks for the fix. I ran into the same problem and using master instead of the latest tagged release fixed the issue. 

However, the clusters no longer respond to cluster events. My cluster events worked in 2.0.9 but since I updated to 2.0.12 and then to master the cluster events no longer work for me.

**Comment by nmccready** [comment]
Can you make an example?

**Comment by normank86** [comment]
Here is a link to a plunker that I made by forking the example from the documentation page for markers: http://plnkr.co/edit/XLYCZV?p=preview . There should be an alert when the cluster is clicked. Also I ran into another bug when I tired to open marker windows. Opening the windows require a double click to open on the first try and then can be opened with a single click there after.

**Comment by adamspe** [comment]
I just ran into the same problem with angular-google-maps v2.0.12.  If I swap out and use http://rawgit.com/angular-ui/angular-google-maps/master/dist/angular-google-maps.js then the problem goes away and when I click on a cluster the underlying markers don't disappear.

**Comment by nmccready** [comment]
Yes I am trying to polish off a few things and put 2.0.13 out soon.

**Comment by ppdg** [comment]
Same issue, on bower v2.0.12.

My clusters disappears when we zoom, but that happen only after the search box integration, that worked well at the beginning. 
I didn't find why, even after having tested the way back. 

Upgrade with master solved the problem :+1: 

Thank you for this work:)

**Comment by nmccready** [comment]
Closing this since it is resolved on master (tba 2.0.13)

**Comment by alcfeoh** [comment]
I have this problem with version 2.3.1. About 30% of the time, if the data is refreshed and the zoom changes, the clusters are gone. They show up once the map is dragged a little. Any ideas?

**Comment by nmccready** [comment]
no please make an example

**Comment by nmccready** [comment]
@alcfeoh please make a new issue
